src/admin/bundles/bundle.routes.js

```diff
@@ -40,7 +40,7 @@
         .state('bundles.bundleSettings', {
             url: 'bundleSettings/:bundleId',
             ncyBreadcrumb: {
-                label: 'admin.manageModuleSettings'
+                label: 'admin.settings'
```

Comment:
This breadcrumb fits better to admin settings page. Exists some reasons to use this?